Angular/login/src/app/components/exchange/exchange.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ibmpariwesh/socgen-stockmarket-project/pullRequest?prid=8405403)
```suggestion
import { Component, OnInit } from "@angular/core";
```